packages/appstore/src/modules/trading-hub/index.tsx

```diff
@@ -1,7 +1,9 @@
 import * as React from 'react';
+import Onboarding from 'Components/onboarding';
+import { trading_hub_contents } from 'Constants/trading-hub-content';
 
 const TradingHub = () => {
-    return <div className='trading-hub'>Trading Hub</div>;
+    return <Onboarding contents={trading_hub_contents} />;
```

Comment:
We need to show this component whenever user click on tour guide button,  so please add a todo here for this